src/components/LoginModal/constants.ts

```diff
@@ -1,5 +1,25 @@
+import { IPopUpTitle } from '../PopUp/PopUpTitle'
+
 export const LOGIN_PAGES = {
   SIGN_IN: 0,
   SIGN_UP: 1,
   FORGOT_PASSWORD: 2,
 }
+
+// This should return localised string keys, e.g. `loginModal.signUp.title` once i18n is
+// implemented. For now we're hardcoding strings.
+export const PAGE_TITLES: IPopUpTitle[] = [
```

Comment:
Are you going to be adding `../PopUp/PopUpTitle`? I don't see it in the files pushed up.

Reply:
Good catch, removed.